.gitignore

```diff
@@ -17,6 +17,7 @@ pickle-email-*.html
 config/initializers/secret_token.rb
 config/secrets.yml
 config/application.yml
```

Comment:
I think you should also remove this line from gitignore

Reply:
Yes you are right. I left it there in case someone upgraded their app with the base app and suddenly they revealed their credentials. But I guess that's not likely to happen, so I'll remove it 👌

Reply:
oh, I see, if that's the case, feel free to keep/remove it.